plugins/paloalto_wildfire/help.md

```diff
@@ -69,7 +71,8 @@ Example input:
 
 |Name|Type|Required|Description|
 |----|----|--------|-----------|
-|submission|filedata|True|Submission|
+|submission|filedata|False|Submission|
```

Comment:
Why is this not a mandatory field anymore?

Reply:
Because as you could see was added verdict field which store information about submitted url. This information could appear when url was previously submitted by someone else. So we do not send url one one more time to API. Just get the verdict information which is stored in API server. If URL do not appear previously we will get submission object after submission :)

Reply:
ok.